### PR TITLE
bandaid the httpoxy vulnerability

### PR DESCRIPTION
**Pull request by millidavids**

### CODE REVIEW

**Comment by millidavids** [comment]
The initial bandaid recommended was to rewrite the HTTP_PROXY header. I'll continue to look into better fixes.

File: infrastructure/nginx/default.conf [hunk 1]
```diff
@@ -30,10 +30,11 @@ server {
         fastcgi_param           QUERY_STRING    $query_string;
         fastcgi_param           CONTENT_TYPE    $content_type;
         fastcgi_param           CONTENT_LENGTH  $content_length;
+        fastcgi_param           HTTP_PROXY "";
```
Comment (bkuhl) [inline infrastructure/nginx/default.conf]: Can we add a comment explaining why this is here?  Maybe a link to the exploit details or something

Comment (jlaswell) [inline infrastructure/nginx/default.conf]: I think a reference to the CSR number is good enough even.

**Comment by jlaswell** [comment]
👍

**Pull request merged**